src/utils/resolveRTL.js

```diff
@@ -0,0 +1,29 @@
+import { from as flatten } from 'array-flatten';
+import rtlCSSJS from 'rtl-css-js';
+
+import separateStyles from './separateStyles';
+
+// Styles is an array of properties returned by `create()`, a POJO, or an
+// array thereof. POJOs are treated as inline styles. This version of the
+// resolve function explicitly does no work to flip styles for an RTL context.
```

Comment:
nit: this comment seems incorrect about RTL

Reply:
Good call!